akka-docs/src/main/paradox/persistence.md

```diff
@@ -1105,6 +1105,170 @@ For more advanced schema evolution techniques refer to the @ref:[Persistence - S
 
 ## Testing
 
+### Persistence TestKit
```

Comment:
I wonder if 'testing' now deserves its own page

Reply:
agree, I was planning to add the examples for Akka Typed.